pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -557,6 +557,7 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				authorizationapi.NewRule("get").Groups(imageGroup).Resources("imagestreamimages", "imagestreams/secrets").RuleOrDie(),
 				authorizationapi.NewRule("get", "update").Groups(imageGroup).Resources("imagestreams").RuleOrDie(),
 				authorizationapi.NewRule("create").Groups(imageGroup).Resources("imagestreammappings").RuleOrDie(),
+				authorizationapi.NewRule("update").Groups(imageGroup).Resources("images").RuleOrDie(),
```

Comment:
Or rather move the `images` entry from line 556 here and extended the verbs for `get` and `update` so resources aren't duplicated in different rules.